io.catenax.request_for_quotation/2.0.0/RequestForQuotation.ttl

```diff
@@ -0,0 +1,577 @@
+
+:rfqId a samm:Property ;
+   samm:preferredName "RFQ identifier"@en ;
+   samm:description "RFQ-Identification: internal identifier in user account"@en ;
+   samm:characteristic :IdCharacteristic ;
+   samm:exampleValue "Drehteil_02_0815" .
+
+:rfqName a samm:Property ;
+   samm:preferredName "RFQ name"@en ;
+   samm:description "RFQ-Identification: name for the request for user account management (usually name of the product to be manufactured)"@en ;
+   samm:characteristic :NameCharacteristic ;
+   samm:exampleValue "Drehteil" .
+
+:rfqDateTime a samm:Property ;
+   samm:preferredName "RFQ date time"@en ;
+   samm:description "RFQ-Identification: date and time of the request in user account"@en ;
+   samm:characteristic :DateTimeCharacteristic ;
+   samm:exampleValue "2023-10-24T14:48:54.709Z"^^xsd:dateTime .
+
+:rfqVersion a samm:Property ;
+   samm:preferredName "RFQ version"@en ;
+   samm:description "RFQ-Identification: version of this request in case there were any changes"@en ;
+   samm:characteristic :RFQVersionCharacteristic ;
+   samm:exampleValue "1.0.0" .
+
+:cadFile a samm:Property ;
+   samm:preferredName "cad file"@en ;
+   samm:description "RFQ-Configuration: CAD file as geometry definition, e.g. step file"@en ;
+   samm:characteristic :FileCharacteristic .
+
+:parts a samm:Property ;
+   samm:preferredName "parts"@en ;
+   samm:description "RFQ-Configuration: list of parts contained in the CAD file"@en ;
+   samm:characteristic :SetOfPartsCharacteristic .
+
+:firstDeliveryDate a samm:Property ;
+   samm:preferredName "first delivery date"@en ;
+   samm:description "RFQ-Configuration: Date of desired, first, but not must delivery"@en ;
+   samm:characteristic :DeliveryDateCharacteristic ;
+   samm:exampleValue "2023-10-24"^^xsd:date .
+
+:lastDeliveryDate a samm:Property ;
+   samm:preferredName "last delivery date"@en ;
+   samm:description "RFQ-Configuration: date of required and last acceptable delivery"@en ;
+   samm:characteristic :DeliveryDateCharacteristic ;
+   samm:exampleValue "2023-12-24"^^xsd:date .
+
+:additionalComments a samm:Property ;
+   samm:preferredName "additional comments"@en ;
+   samm:description "RFQ-Configuration: placeholder for any kind of additional comments"@en ;
+   samm:characteristic :CommentCharacteristic ;
+   samm:exampleValue "this is a prototype, recommendations towards design for manufacturing are highly welcome" .
+
+:additionalFiles a samm:Property ;
+   samm:preferredName "additionalFiles"@en ;
+   samm:description "RFQ-Configuration: in case there are additional documents or files to share (like e.g. technical drawings)"@en ;
+   samm:characteristic :SetOfFilesCharacteristic .
+
+:orderQuantity a samm:Property ;
+   samm:preferredName "order quantity"@en ;
+   samm:description "RFQ-Configuration: quantity at all, valid for complete order (usually number of products to be ordered)"@en ;
+   samm:characteristic :OrderQuantityCharacteristic .
```

Comment:
for quantity, please check if you can make use of existing shared [quantity model ](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.shared.quantity/1.0.0/Quantity.ttl), then you don't need to define your own Characteristic

Reply:
Thanks for the hint! The quantity value characteristic has been reused

Reply:
are you able to use the `ext-quantity` here also or not?